cmake/SetupCodeChecks.cmake

```diff
@@ -127,11 +234,11 @@ macro(blt_add_uncrustify_target)
             WORKING_DIRECTORY ${_wd} 
             COMMENT "${arg_COMMENT}Running uncrustify source code formatting checks.")
```

Comment:
Should there be a space between ``${arg_COMMENT}`` and ``Running`` ?

Reply:
If you put a space there when there isn't a comment it has an extra space at the beginning.